Readme.md

```diff
@@ -5,13 +5,13 @@
   [![build status][travis-image]][travis-url]
   [![Test coverage][coveralls-image]][coveralls-url]
 
-  Expressive middleware for node.js using generators via [co](https://github.com/visionmedia/co)
-  to make web applications and APIs more enjoyable to write. Koa's middleware flow in a stack-like manner allowing you to perform actions downstream, then filter and manipulate the response upstream. Koa's use of generators also greatly increases the readability and robustness of your application.
+  Expressive HTTP middleware for node.js, using generators via [co](https://github.com/visionmedia/co),
```

Comment:
i'd prefer to remove the entire "using generators via [co](https://github.com/visionmedia/co)" phrase because it's just implementation detail. @koajs/owners what do you guys think?